applications/openshift/kubelet/var_kubelet_evictionsoft_imagefs_available.var

```diff
@@ -11,7 +11,7 @@ operator: equals
 interactive: false
 
 options:
-  default: "15%"
+  default: "20%"
```

Comment:
same question about all the soft parameters - I see all of them still failing, why are we changing the defaults?

Reply:
Because the default hard eviction values should be less than the soft eviction values